gio/src/subclass/file_monitor.rs

```diff
@@ -0,0 +1,56 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
```

Comment:
Given that this one is very simple, we can review/merge it in a separate PR already.